### PR TITLE
You can monkeyize Lizards into Kobolds now

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Approved by @Mothblocks 

Really the title says it all. They're a lot more monkeylike than they are lizardlike.

One thing I should say is that they don't have a special language for themselves. They speak draconic (which means lizards will be able to speak to them) but they can't even understand common to balance that out.

- [ ] finish one last overlay

## Why It's Good For The Game

Monkified is a super cool gene, I think players will enjoy expanding the possibilities here.

## Changelog
:cl: Mey Ha Zah, Bazelart
add: You can monkify lizards now. Dios Mios...
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
Not a particular fan of this, I don't think we need more sub species

**Comment by ATH1909** [comment]
> but unfortunately between monkified being locked to humans and plasmamen being dna immune, the meta is human only if you're playing geneticist which sucks.

you can monkify moths and such; it's not a human-only mutation

**Comment by tralezab** [comment]
I'll edit OP

File: code/datums/mutations/body.dm [hunk 1]
```diff
@@ -169,11 +169,11 @@
...
 
 
-//Monified turns you into a monkey.
+//Primal turns you into a lesser form of your species, if your species has that.
 /datum/mutation/human/race
-	name = "Monkified"
-	desc = "A strange genome, believing to be what differentiates monkeys from humans."
-	text_gain_indication = "You feel unusually monkey-like."
+	name = "Primal"
+	desc = "A strange genome, holding a \"genetic memory\" of the primal ancestors of this species. In case of activation, a backward evolution process will trigger."
```
Comment (ATH1909) [inline code/datums/mutations/body.dm:175]: does this mean that the monkeys from genetics were actually originally humans who were forcibly turned into monkeys?

File: code/modules/mob/living/carbon/human/species_types/kobolds.dm [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	liked_food = GROSS | MEAT
+	limbs_id = "kobold"
+	bodypart_overides = list(
+		BODY_ZONE_L_ARM = /obj/item/bodypart/l_arm,\
+		BODY_ZONE_R_ARM = /obj/item/bodypart/r_arm,\
+		BODY_ZONE_HEAD = /obj/item/bodypart/head,\
+		BODY_ZONE_L_LEG = /obj/item/bodypart/l_leg,\
+		BODY_ZONE_R_LEG = /obj/item/bodypart/r_leg,\
+		BODY_ZONE_CHEST = /obj/item/bodypart/chest)
+	species_language_holder = /datum/language_holder/kobold
```
Comment (ATH1909) [inline code/modules/mob/living/carbon/human/species_types/kobolds.dm:21]: don't forget to give them kobolds `ass_image = 'icons/ass/asslizard.png'`

Comment (ATH1909) [reply code/modules/mob/living/carbon/human/species_types/kobolds.dm:21]: and the body temperature stuff

File: code/modules/mob/living/carbon/human/species_types/monkeys.dm [hunk 3]
```diff
@@ -35,6 +35,7 @@
 	gib_anim = "gibbed-m"
 
 	payday_modifier = 1.5
+	var/can_gorrilize = TRUE
```
Comment (ATH1909) [inline code/modules/mob/living/carbon/human/species_types/monkeys.dm:38]: you forgot to set this to FALSE for kobolds

note to self: make some sort of dragon-like creature the gorilla equivalent for kobolds

File: code/modules/surgery/organs/tails.dm [hunk 4]
```diff
@@ -105,3 +105,24 @@
 		H.dna.features["tail_monkey"] = "None"
 		H.dna.species.mutant_bodyparts -= "tail_monkey"
 		H.update_body()
+
+/obj/item/organ/tail/lizard/kobold
```
Comment (ATH1909) [inline code/modules/surgery/organs/tails.dm:109]: don't forget to add a limbgrower design for a /obj/item/organ/tail/lizard/kobold/fake subtype of these

**Comment by ATH1909** [comment]
now I (or someone else; feel free to copy the gimmick) can play as a kobold with a wand of death when I next roll wizard

File: code/modules/surgery/organs/tails.dm [hunk 5]
```diff
@@ -105,3 +105,24 @@
...
+	name = "kobold tail"
+	desc = "A smaller version of a normal lizard tail. Somewhere, no doubt, a lizard hater feels cheated."
+	tail_type = "Kobold"
+	icon_state = "severedkoboldtail"
+
+/obj/item/organ/tail/monkey/Insert(mob/living/carbon/human/H, special = FALSE, drop_if_replaced = TRUE)
```
Comment (Fikou) [inline code/modules/surgery/organs/tails.dm:115]: wrong path

File: code/modules/surgery/organs/tails.dm [hunk 6]
```diff
@@ -105,3 +105,24 @@
...
+	..()
+	if(istype(H))
+		if(!("tail_kobold" in H.dna.species.mutant_bodyparts))
+			H.dna.species.mutant_bodyparts |= "tail_kobold"
+			H.dna.features["tail_kobold"] = tail_type
+			H.update_body()
+
+/obj/item/organ/tail/monkey/Remove(mob/living/carbon/human/H, special = FALSE)
```
Comment (Fikou) [inline code/modules/surgery/organs/tails.dm:123]: ditto

File: code/modules/mob/living/carbon/human/species_types/kobolds.dm [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+	disliked_food = GRAIN | DAIRY | CLOTH
+	liked_food = GROSS | MEAT
+	limbs_id = "kobold"
+	bodypart_overides = list(
+		BODY_ZONE_L_ARM = /obj/item/bodypart/l_arm,\
+		BODY_ZONE_R_ARM = /obj/item/bodypart/r_arm,\
+		BODY_ZONE_HEAD = /obj/item/bodypart/head,\
+		BODY_ZONE_L_LEG = /obj/item/bodypart/l_leg,\
+		BODY_ZONE_R_LEG = /obj/item/bodypart/r_leg,\
+		BODY_ZONE_CHEST = /obj/item/bodypart/chest)
```
Comment (Fikou) [inline code/modules/mob/living/carbon/human/species_types/kobolds.dm:20]: the monkey bodyparts have some overrides you might be overlooking like lesser wound resistance or the animal_origin thing

**Comment by Serxule** [comment]
> Not a particular fan of this, I don't think we need more sub species

oranges. let us have fun. stop trying to remove everything fun in the game

**Comment by Farquaar** [comment]
What do kobolds look like in-game?

**Comment by maxymax13** [comment]
> @tgstation temporarily blocked @Serxule 
:trollface:

**Comment by optimumtact** [comment]
> > @tgstation temporarily blocked @Serxule
> 
> :trollface:

please don't comment spam, just use the emojii. save comments for detailed feedback or review of the code.

**Comment by Mey-Ha-Zah** [comment]
I had a lot of fun making the Kobold, really hope this gets accepted.

**Comment by Time-Green** [comment]
> What do kobolds look like in-game?

This. The iconbot just shows it deconstructed, we wanna see the REAL kobolds

**Comment by Mothblocks** [comment]
Haven't looked into this yet, but what I okayed was more or less the equivalent to chickens for Vox (where they are monkified into, more or less, simple mobs). Will need to see the scope of this.

File: code/modules/surgery/organs/tails.dm (hunk 5 above)
Comment (Time-Green) [reply code/modules/surgery/organs/tails.dm:115]: Kobolds are humans though? Seems like the correct path. Also you should use /human insteaf of /H, no idea why but maintainers demand it

**Comment by Mothblocks** [comment]
Yeah this looks alright, my only concern is that eventually people will take this from a reskin into something a lot more materially different, which I don't approve of, so will just need to make sure we put our foot down when that happens.

File: code/modules/surgery/organs/tails.dm (hunk 5 above)
Comment (Mothblocks) [reply code/modules/surgery/organs/tails.dm:115]: he meant that its tail/monkey

File: code/modules/surgery/organs/tails.dm (hunk 4 above)
Comment (Mothblocks) [reply code/modules/surgery/organs/tails.dm:109]: please stop suggesting extra features to prs

Comment (Mothblocks) [inline code/modules/surgery/organs/tails.dm:109]: Does this need to be a subtype of lizard tails? What benefit does that give us?

**Comment by Mothblocks** [comment]
To be clear--my goal is that these are, ideally, just monkey reskins. I don't really want any non-cosmetic behavior aside from that.

**Comment by Time-Green** [comment]
> To be clear--my goal is that these are, ideally, just monkey reskins. I don't really want any non-cosmetic behavior aside from that.

CATERPILLAR MOTH MONKEYS

**Comment by Sparkezel** [comment]
Pictures please

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 8]
```diff
@@ -1562,6 +1562,12 @@
 	icon_state = "lbelly"
 	gender_specific = 1
 
+//there was no other way... i'm sorry.......
+/datum/sprite_accessory/diaper/default
```
Comment (Mothblocks) [inline code/modules/mob/dead/new_player/sprite_accessories.dm:1566]: i dont know if i can approve of this on moral grounds anymore

**Comment by moocowswag** [comment]
If they are supposed to be nothing more than a monkey reskin will they turn into gorillas when irradiated?

**Comment by RaveRadbury** [comment]
> Haven't looked into this yet, but what I okayed was more or less the equivalent to chickens for Vox (where they are monkified into, more or less, simple mobs). Will need to see the scope of this.

We do have lizards already

**Comment by Mothblocks** [comment]
yeah thats true but both are cute to me so its alrigth

**Comment by tralezab** [comment]
You guys will get that picture trust me

**Comment by Ryll-Ryll** [comment]
I'm confused, these look nothing like the canonical dnd kobolds

image

**Comment by davethwave** [comment]
There are typically two different kind of kobold. One the the dog people and the other are the dragon worshipping lizards the PR guy made here.

**Comment by Ryll-Ryll** [comment]
Oh I know, I just wanted an excuse to post the dog kobold because god just look at it

**Comment by NullFag** [comment]
> Not a particular fan of this, I don't think we need more sub species

Kind of same opinion here.
Besides, weren't lizardpeople supposed to be the lesser species already? Why make a lesser lesser specie?

**Comment by Mothblocks** [comment]
monkey

**Comment by TheBonded** [comment]
> > Not a particular fan of this, I don't think we need more sub species
> 
> Kind of same opinion here.
> Besides, weren't lizardpeople supposed to be the lesser species already? Why make a lesser lesser specie?

You could say the same thing about felinds

**Comment by tralezab** [comment]
Lesser species only in terms of balance, humans should be the "meta pick"

**Comment by optimumtact** [comment]
I've thought about this more and I have come to the inescapable conclusion that this adds nothing of value

**Comment by Mothblocks** [comment]
ah well

**Comment by tralezab** [comment]
I walk a lonely road
The only one that I have ever known
Don't know where it goes
But it's home to me, and I walk alone
I walk this empty street
On the Boulevard of Broken Dreams
Where the city sleeps
And I'm the only one, and I walk alone

**Comment by bluetshirtguy** [comment]
sad

**Comment by NullFag** [comment]
> You could say the same thing about felinds

These don't exist and you know it.

**Comment by ThePooba** [comment]
@tralezab Hey, what do you think about monkestation (TG downstream) adding this?

**Pull request closed without merging**